charts/kyverno/values.yaml

```diff
@@ -143,6 +143,24 @@ envVars: {}
 extraArgs:
   - --autogenInternals=false
 
+# -- Array of extra init containers
+#
+# Example:
```

Comment:
Put the example after the default value.  Otherwise the README ends up with example in description.

Example:

```
| extraInitContainers | list | `[]` | Array of extra init containers Example: - name: init-container   image: busybox   command: ['sh', '-c', 'echo Hello'] |
```

Reply:
isn't this what i did?

Reply:
Yep , this is fixed.